doc/settings.md

```diff
@@ -19,6 +19,30 @@ See [details on telemetry](https://github.com/microsoft/winget-create#datateleme
 
 If set to true, the `telemetry.disable` setting will prevent any event from being written by the program.
 
+## CleanUp
+
+The `CleanUp`` settings determine whether Winget-Create will handle the removal of temporary files (installer cache and logs) generated during the manifest creation process. These settings provide control over the decision to remove files or not and the frequency at which this clean up occurs.
+
+### disable
+
+```json
+    "CleanUp": {
+        "disable": true
```

Comment:
Is this actually needed? Couldn't a user just set the intervalInDays to `-1` to effectively disable the cleanup?

Reply:
Had that in mind but decided against it since I thought that wouldn't immediately be obvious to the user opening the settings in VSCode and would probably require referring to the documentation. I'm fine either way, I'll wait if @ryfu-msft agrees with the suggestion and I'll update the schema

Reply:
I was just curious more than anything. Doesn't really make a difference either way

Reply:
I think having a boolean that explicitly enables/disables this feature is probably the safest route. However, I do think non-default behavior should always be set to false. 

In other words, I think this should be changed to `CleanUp.enable = false` as the default case.

Reply:
That's a good point. I am okay with making it a default behavior as I'd rather have proper cleanup to save space for our users so they're not storing a bunch of installers.